.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.20
```

Comment:
Is this just an `asdf`-related file, or does it somehow impact the CI/CD process?

I'm wondering if it's necessary to have it in the repo, or if it's specific to your individal dev env. If we want to keep it here, it might be worth adding a comment to the file to explain what its for and when/where it applies.